SPECS/xdp-tools/xdp-tools.spec

```diff
@@ -0,0 +1,157 @@
+Summary:        Utilities and example programs for use with XDP
+Name:           xdp-tools
```

Comment:
Thank you for your contribution.

Initial comments:

- We are missing a mandatory `xdp-tools.signature.json` file; which, if absent will break the build :(
- We need to upload the source `tar.gz` to the `blobstore` 

I have un-checked above items from the checklist in the PR.

Please feel free to reach-out in case you need any help :)

Reply:
so i need to move this from SPECS to SPECS-EXTENDED and generate the signature.json, right?

Reply:
what is this blobstore? do i have access to it?

Reply:
i've moved the specfile over to `SPECS-EXTENDED` and added a .signature.json. still unsure about this blobstore.

Reply:
I went ahead and uploaded the source tarball to our blobstore. Quick explanation- our SRPM packing tool doesn't pull sources directly from the source URL. Instead, we archive all sources and make them available in a known location (`cblmarinerstorage.blob.core.windows.net/sources/core`). That way, we don't ever have to deal with build breaks from source URLs going offline.